cmd/goa/gen.go

```diff
@@ -142,16 +142,34 @@ func (g *Generator) Write(debug bool) error {
 	return err
 }
 
+const GOMODENVKEY = "GOMOD"
+
+func findGoMod() (string, error) {
```

Comment:
Looks like this function can never return an error which is good so the error return value can be removed.